glite-info-update-endpoints.spec

```diff
@@ -66,10 +66,10 @@ rm -rf %{buildroot}
 - BUG #96484: Fixed post install actions
 - BUG #97395: Fixed rpmlint errors
 
-* Mon May 25 2012 Laurence Field <laurence.field@cern.ch> - 2.0.10-1
```

Comment:
Do we hate [RFC 3339](https://tools.ietf.org/html/rfc3339) ? 
This makes the changelog difficult for machines to read, doesn't it?

Reply:
Yes, but I think it's the default date format in RPMS, not sure if we can change it...